tests/log_printfnoformat/Makefile

```diff
@@ -0,0 +1,10 @@
+include ../Makefile.tests_common
+
+USEMODULE += log_printfnoformat
+
+TEST_ON_CI_WHITELIST += all
+
+# Enable debug log level
+CFLAGS += -DLOG_LEVEL=4
```

Comment:
Idea for future "feature": make `LOG_LEVEL` configurable via global variable similar to `DEVELHELP`.

Reply:
Hm, what would be the advantage over just setting the define?

Reply:
Ease of use.

Reply:
> make LOG_LEVEL configurable via global variable similar to DEVELHELP

I like the idea :)